connectors/katalog/pkg/connector/connector.go

```diff
@@ -95,30 +100,36 @@ func (r *Handler) createAsset(c *gin.Context) {
 		return
 	}
 
+	errString := "Error during create asset! Error:"
+	secretName, secretNamespace, err := vault.GetKubeSecretDetailsFromVaultPath(request.Credentials)
+	if err != nil {
+		r.reportError(c, http.StatusInternalServerError, errString+err.Error())
```

Comment:
Use errors.Wrap to add the message to the error

Reply:
Fixed Thanks